ec2/Vagrantfile

```diff
@@ -100,7 +100,8 @@ if ['up', 'ssh', 'destroy', 'halt'].include?(ARGV[0])
         raise "#{ENV['HOME']}/.okta_aws.toml is not defined, cannot continue. Please read README.md for an example and usage details."
       end
       puts " * You might be prompted for your Okta password now..."
-      `okta_aws "#{aws_profile}"`
+      #`okta_aws "#{aws_profile}"`
+      `saml2aws login --profile chef-engineering`
```

Comment:
why is this change needed in Milestone Version work ?

Reply:
technically, these changes no way related to milestone changes, while setting up the ec2 for testing, we modified this file as above, and since these changes are needed for setting up ec2 using vagrant, we haven't reverted the changes.